config.json

```diff
@@ -1185,6 +1199,6 @@
     }
   ],
   "foregone": [
-
+    
```

Comment:
@rodrigocam, the extra spaces here also need to be removed. There should be a blank line, but there should be nothing in it.